projects/packages/forms/src/blocks/contact-form/class-contact-form-block.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Subscriptions Block.
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Contact_Form;
+
+use Automattic\Jetpack\Assets;
+use Automattic\Jetpack\Blocks;
+use Automattic\Jetpack\Forms\ContactForm\Contact_Form;
+
+/**
+ * Contact Form block render callback.
```

Comment:
I think I would go into more details here, to explain the overall infrastructure of the block in this package:

- A parent block in its own class
- Child blocks loaded separately (maybe they should be part of this class too, in their own method?)
- How a plugin can load those different items on their end.

Basically, I think we would benefit from a how-to in here so one can easily load the contact form block in their own plugin in the future.